javatests/arcs/android/integration/policy/scenarios.arcs

```diff
@@ -0,0 +1,40 @@
+// Copyright 2021 Google LLC.
+//
+// This code may only be used under the BSD style license found at
+// http://polymer.github.io/LICENSE.txt
+//
+// Code distributed by Google as part of this project is also subject to an additional IP rights
+// grant found at
+// http://polymer.github.io/PATENTS.txt
+
+meta
+  namespace: arcs.android.integration.policy
+
+import './particles.arcs'
+
+/// Scenario: A policy-compliant recipe with @persistent handles egresses data
+/// and stores ingress-restricted values to disk that is never deleted
+
+@egressType('GeneralEgress')
+policy SimplePersistencePolicy {
+  @allowedRetention(medium: 'Disk', encryption: false)
+  from Thing access {
+    b,
+    @allowedUsage(label: 'raw', usageType: '*')
+    c
+  }
+}
+
+// Given a recipe is compliant with a policy
```

Comment:
What happened to the scenario comment you had earlier? OK to have removed it if you thought the comment did not add value.

Reply:
I removed them here and updated the comments in the test file.